lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -702,7 +726,7 @@ Expr RawMemManager::ptrtoint(PtrTy ptr, const Type &ptrTy,
 PtrTy RawMemManager::gep(PtrTy ptr, gep_type_iterator it,
                          gep_type_iterator end) const {
   Expr offset = m_sem.symbolicIndexedOffset(it, end, m_ctx);
-  return offset ? ptrAdd(ptr, offset) : Expr();
+  return offset ? ptrAdd(ptr, offset) : m_nullPtr;
```

Comment:
very very different behavior. Previously used to return null that is interpreted as non-deterministic, when offset was not computed. Now returns a symbolic `nullptr`

Reply:
Good point. Fixed.